includes/data-sources-sdks.md

```diff
@@ -7,9 +7,10 @@
 - :javascript-color: [JavaScript](../data/sdks/javascript/index.md)
```

Comment:
Maybe we tag this as legacy and move it down in the order, and bring up `Browser`.

So `JavaScript (Legacy)` ?

Generally this list should be ordered by what we want the customer to use. (or even just remove JavaScript...?)

Reply:
I think we should just remove it. Otherwise at one point it's be confusing to see all new and legacy together